### PR TITLE
feat - made the home page

### PR DESCRIPTION
**Pull request by therajatg**

### CODE REVIEW

File: my-app/src/frontend/pages/home/Home.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+import {Filter} from '../.././components/filter/Filter'
+import {Footer} from '../.././components/footer/Footer'
 
 
 function Home() {
   return (
     <div>
-        <Navbar/>
-        <Products/>
+      <Navbar/>
```
Comment (rahulcs754) [inline my-app/src/frontend/pages/home/Home.js]: you can use this navbar to app js.  
if you use navbar component in app js then you will not need to call every page

**Review by rahulcs754 — APPROVED** [review]
code looks good. 
Just Small suggestion
- remove extra space
- you can add nav component to app js . after that you will not need to import every component

Your design is looks good

File: my-app/src/frontend/pages/home/Home.js (hunk 1 above)
Comment (therajatg) [reply my-app/src/frontend/pages/home/Home.js]: great suggestion man. The navabar and footer both can render only one time and not with every page. 

Made the suggested changes.

File: my-app/src/frontend/components/products/Products.js [hunk 2]
```diff
@@ -1,7 +1,44 @@
...
+  const [productListing, setProductListing] = useState([]);
+  useEffect(() => {
+    fetchData();
+  }, [])
+
+const fetchData = async () => {
+    try {
+      const response = await axios.get("/api/products");
+      // const d = await response.json(); (If I do this then catch is shown in console)
+      console.log(response.data.products);
```
Comment (Mohitkumar33) [inline my-app/src/frontend/components/products/Products.js:17]: You can remove console.log

**Review by Mohitkumar33 — APPROVED** [review]
code looks clean. You can use formatter (prettify) to beautify the code good. And you can change that in setting that auto format on save that will help you

File: my-app/src/frontend/components/products/Products.js (hunk 2 above)
Comment (therajatg) [reply my-app/src/frontend/components/products/Products.js:17]: will do

**Pull request merged**